packages/graphql/src/utils/error-has-gql-status.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Neo4jError } from "neo4j-driver";
+
+export function errorHasGQLStatus(error: Neo4jError, gqlStatus: string): boolean {
+    if (error.gqlStatus === gqlStatus) {
+        return true;
+    }
+
+    if (error.cause) {
+        if (!(error.cause instanceof Neo4jError)) {
```

Comment:
Not 100% sure that the wrapped error will have be a Neo4jError

Reply:
tbh I'm not sure, but the typings just have an error here